connectionchecker/connection_checker_test.go

```diff
@@ -59,12 +59,27 @@ func TestConnectionChecker(t *testing.T) {
 		},
 
 		{
+			name: "MongoDB with no auth",
+			req: &agentpb.CheckConnectionRequest{
+				Dsn:     "mongodb://127.0.0.1:27018/admin?connectTimeoutMS=1000",
```

Comment:
Let's add another test: with credentials for DB that does not require them

Reply:
added